### PR TITLE
Support .NET 4.8 for UI packages

### PR DESCRIPTION
**Pull request by Shane32**
Notes:
- ASP.NET 2.1 contains a reference to Newtonsoft.Json, while .NET Core 3.1 includes System.Text.Json.  So either way there is a serializer available without an additional reference.
- There are no ifdefs for the net5 and net6 TFMs, so they have been removed.

### CODE REVIEW

File: src/Ui.Altair/Internal/AltairPageModel.cs [hunk 1]
```diff
@@ -37,11 +36,20 @@ public string Render()
...
 
             _altairCSHtml = _options.PostConfigure(_options, builder.ToString());
         }
 
         return _altairCSHtml;
     }
+
+    private static string JsonSerialize(object value)
+    {
+#if !NETSTANDARD2_0
```
Comment (sungam3r) [inline src/Ui.Altair/Internal/AltairPageModel.cs:49]: I would invert condition.

**Comment by sungam3r** [comment]
> There are no ifdefs for the net5 and net6 TFMs, so they have been removed.

I would prefer to have explicit tfms even without ifdefs.

**Comment by sungam3r** [comment]
Nice and simple change.

File: src/Ui.Altair/Internal/AltairPageModel.cs (hunk 1 above)
Comment (Shane32) [reply src/Ui.Altair/Internal/AltairPageModel.cs]: Ok

**Comment by Shane32** [comment]
> I'm a bit concerned about restoring 3.1 tfm from 5.0/6.0 apps.

Please define your concern and I will address it.  There is no difference here between targeting .NET Standard 2.0 or 2.1, except we have defined a minimum baseline of .NET Core 3.1.  Going forward there is no .NET Standard, which is there are ifdefs like NETCOREAPP3_1_OR_GREATER.  There is no need to target every newly released .NET Core version, and we do not do so in any of our other published packages.

File: src/Ui.Altair/Internal/AltairPageModel.cs (hunk 1 above)
Comment (Shane32) [reply src/Ui.Altair/Internal/AltairPageModel.cs]: Btw, you have requested this change many times, and I do not agree with the general principle that conditions should be inverted if they have a `!`.  In this instance the ifdef would have matched the other ifdef in the project, and would place the typical/default/primary code first.  I think you should reconsider your viewpoint.  Regardless, I changed it as you requested.

Comment (sungam3r) [reply src/Ui.Altair/Internal/AltairPageModel.cs]: I reason from the simplicity of reading code when you do not mentally invert condition even if typical/default/primary code implies `!`.

**Comment by sungam3r** [comment]
Something like phantom fears.

**Review by sungam3r — APPROVED** [review]

**Pull request merged**